packages/frontend/@n8n/design-system/src/components/AskAssistantChat/messages/TextMessage.vue

```diff
@@ -0,0 +1,202 @@
+<script setup lang="ts">
+import Markdown from 'markdown-it';
+import markdownLink from 'markdown-it-link-attributes';
+import { computed } from 'vue';
+
+import BaseMessage from './BaseMessage.vue';
+import { useI18n } from '../../../composables/useI18n';
+import type { ChatUI } from '../../../types/assistant';
+import BlinkingCursor from '../../BlinkingCursor/BlinkingCursor.vue';
+
+interface Props {
+	message: ChatUI.TextMessage & { id: string; read: boolean; quickReplies?: ChatUI.QuickReply[] };
+	isFirstOfRole: boolean;
+	user?: {
+		firstName: string;
+		lastName: string;
+	};
+	streaming?: boolean;
+	isLastMessage?: boolean;
+}
+
+defineProps<Props>();
+const { t } = useI18n();
+
+const md = new Markdown({
```

Comment:
These markdown configurations are repeated a few times in message components. Can we centralize them?